src/tests/get-users-test.ts

```diff
@@ -92,16 +110,75 @@ describe('Testing get users query', () => {
     const response = await axios.post(
       `http://localhost:${process.env.PORT}`,
       {
-        query: `
-        query Users($numberOfUsers: Int) {
-          users(numberOfUsers: $numberOfUsers) {
-            id
-            name
-            email
-            birthDate
-          }
-        }
-      `,
+        query: print(getManyUsers),
+        variables: {
+          data: {
+            numberOfUsers: 5,
+            skippedUsers: 0,
+          },
+        },
+      },
+      {
+        headers: {
+          authorization: token,
+        },
+      },
+    );
+
+    const expectedResponse = {
+      users: createdUsers,
+      total: 5,
+      hasBefore: false,
+      hasAfter: false,
+    };
+
+    expect(response.data.data.users).to.be.deep.eq(expectedResponse);
+  });
+
+  it('Should return the first 2 users from db', async () => {
```

Comment:
```suggestion
  it('Should return the first page of users', async () => {
```
ao invés de colocar valores e o expect explicito na descrição, seria mais interessante descrever o cenário para uma pessoa. Aqui, por exemplo, descrever que vc está testando a primeira página da chamada (mesma coisa para os demais its de paginação)